<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matrices</class>
 <widget class="QWidget" name="Matrices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Suma y resta de matrices</string>
  </property>
  <widget class="QPushButton" name="Mat_Btt_gToIni">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ir a inicio</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>11</y>
     <width>261</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="MatrizA">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nª filas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="NFilasSpinBoxA"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nº columnas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="NColumnasSpinBoxA"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>11</y>
     <width>258</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableView" name="MatrizB"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nº filas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="NFilasSpinBoxB_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Nª columnas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="NFilasSpinBoxB"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>12</y>
     <width>258</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTableView" name="MatrizResultado"/>
    </item>
    <item>
     <widget class="QPushButton" name="Mat_Btt_SumMat">
      <property name="text">
       <string>Sumar matrices</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Mat_Btt_SumMat_2">
      <property name="text">
       <string>Restar matrices</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Mat_Btt_gToIni_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>refrescar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
